torch/_inductor/constant_folding.py

```diff
@@ -163,11 +167,13 @@ def run(self):
 
 
 @torch.utils._python_dispatch._disable_current_modes()
-def constant_fold(gm):
+def constant_fold(gm, filter_fn: Optional[Callable[[torch.fx.Node], bool]] = None):
```

Comment:
nit `filter_fn` is somewhat ambiguous as in filtering which way, `constraint_fn` might be better i.e only fold if following `constraint` is satisfied. Not a string opinion. Ok with filter_fn too.

Reply:
oh OK